### PR TITLE
Feature/connect to backend

### PR DESCRIPTION
**Pull request by krdurrett**
____ Wrote Tests ___x_ Implemented ____ Reviewed

## Neccesary checkmarks:

    [] All Tests are Passing in all environments

    [x] The code will run locally

## Type of change

    [] New feature
    [] Bug Fix
    [x] Refactor

## Description of Change:

    description: Connected to backend deployed heroku app
    
## Requested feedback: 
    
    I'd like feedback on... n/a

## Check the correct boxes

    [x] This broke nothing
    [] This broke some stuff
    [] This broke everything

## Testing Changes

    [x] No Tests have been changed
    [] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

    [] My code has no unused/commented out code
    [] My code has no binding.pry's
    [] I have reviewed my code
    [] I have commented my code, particularly in hard-to-understand areas
    [] I have fully tested my code

(For Fun!)Please Include a link to a gif of your feelings about this branch

Link:

### CODE REVIEW

File: src/Components/Home.js [hunk 1]
```diff
@@ -26,12 +26,12 @@ const Home = () => {
 
 
   if (loading) return "Loading..."
-
-  if (error) {
-    setError(error)
+  
+  if (data.errors) {
```
Comment (RosaTheDev) [inline src/Components/Home.js:30]: Nice that you solved the issue with placing a return!

File: src/Components/SingleCryptid.js [hunk 2]
```diff
@@ -24,11 +24,6 @@ const SingleCryptid = () => {
   const {data, error, loading} = useQuery(GET_SINGLE_CRYPTID, {
     variables: {
       name: name
-    },
```
Comment (RosaTheDev) [inline src/Components/SingleCryptid.js:27]: We don't need headers when it's HTPP?

**Review by RosaTheDev — APPROVED** [review]

File: src/Components/Home.js [hunk 3]
```diff
@@ -6,8 +6,8 @@ import { ErrorContext } from '../Context/ErrorContext'
 import { useQuery, gql } from "@apollo/client"
 
 const GET_CRYPTIDS = gql`
-    query getCryptids {
-     cryptids {
+    query GetCryptids {
```
Comment (GraceGardner) [inline src/Components/Home.js:9]: We're linked!

**Review by GraceGardner — APPROVED** [review]
Nice work getting linked

**Pull request merged**